modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/lifecycle/DataStreamLifecycleServiceIT.java

```diff
@@ -706,6 +677,35 @@ public void testReenableDataStreamLifecycle() throws Exception {
         });
     }
 
+    private static List<String> getBackingIndices(String dataStreamName) {
```

Comment:
I just moved this helper function in the bottom to be after all the test cases.

Reply:
Thanks for this note, this is really helpful when reviewing ❤️